### PR TITLE
DD 4.15: Dev Envs content

### PR DESCRIPTION
**Pull request by aevesdocker**
PR to document the 4.15 feature work for the Dev Envs team.
- Adds info on the new Environments browser extension
- Users can now choose which IDE they want to use
- Users can now name the dev environment if they wish

### CODE REVIEW

File: desktop/dev-environments/create-dev-env.md [hunk 1]
```diff
@@ -101,23 +107,24 @@ Docker then clones the repository with your specified branch or tag.
...
 ## Launch from a local folder
 
 1. From **Dev Environments** in Docker Dashboard, select **Create**. The **Create a Dev Environment** dialog displays.
-2. Select **Get Started** and then choose **Local Folder** as the source.
-3. Next to **Select your local directory** field, select **Select** to open the root of the code that you would like to work on.
-4. Select **Continue**.
-
-    This detects the main language of your local folder, creates a dev environment using your local folder, and bind-mounts your local code in the dev environment. It then opens VS Code inside the dev environment container.
+2. Select **Get Started**.
+3. Optional: Provide a name for you dev environment.
```
Comment (hyu) [inline desktop/dev-environments/create-dev-env.md:124]:
```suggestion
3. Optional: Provide a name for your dev environment.
```

File: desktop/dev-environments/create-dev-env.md [hunk 2]
```diff
@@ -101,23 +107,24 @@ Docker then clones the repository with your specified branch or tag.
...
+4. Choose **Local directory** as the source.
+5. Select **Select** to open the root of the code that you would like to work on.
```
Comment (hyu) [inline desktop/dev-environments/create-dev-env.md:126]:
```suggestion
5. Select **Select** to open the root directory of the code that you would like to work on.
```

(This read more clearly to me. Somehow "root" without directory made me think about root user or something)

File: desktop/dev-environments/create-dev-env.md [hunk 3]
```diff
@@ -101,23 +107,24 @@ Docker then clones the repository with your specified branch or tag.
...
+    A bind mount is created to share the directory from your computer to the container. You can use an editor or IDE of your choice.
```
Comment (hyu) [inline desktop/dev-environments/create-dev-env.md:127]:
```suggestion
    A directory from your computer is bind mounted to the container, so any changes you make locally will be reflected in the Dev Environment. You can use an editor or IDE of your choice.
```

Clunky phrasing, but I wanted to rephrase "share" which didn't quite seem like the right word here.

**Review by hyu — APPROVED** [review]
LGTM! Found one typo and left a couple comments that you are free to ignore.

File: desktop/dev-environments/create-dev-env.md [hunk 4]
```diff
@@ -101,23 +107,24 @@ Docker then clones the repository with your specified branch or tag.
...
-2. Select **Get Started** and then choose **Local Folder** as the source.
-3. Next to **Select your local directory** field, select **Select** to open the root of the code that you would like to work on.
-4. Select **Continue**.
-
-    This detects the main language of your local folder, creates a dev environment using your local folder, and bind-mounts your local code in the dev environment. It then opens VS Code inside the dev environment container.
+2. Select **Get Started**.
+3. Optional: Provide a name for your dev environment.
+4. Choose **Local directory** as the source.
+5. Select **Select** to open the root directory of the code that you would like to work on.
+    A directory from your computer is bind mounted to the container, so any changes you make locally will be reflected in the Dev Environment. You can use an editor or IDE of your choice.
```
Comment (aevesdocker) [inline desktop/dev-environments/create-dev-env.md:127]:
```suggestion
    A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```

File: desktop/dev-environments/create-dev-env.md [hunk 5]
```diff
@@ -101,23 +107,24 @@ Docker then clones the repository with your specified branch or tag.
...
-2. Select **Get Started** and then choose **Local Folder** as the source.
-3. Next to **Select your local directory** field, select **Select** to open the root of the code that you would like to work on.
-4. Select **Continue**.
-
-    This detects the main language of your local folder, creates a dev environment using your local folder, and bind-mounts your local code in the dev environment. It then opens VS Code inside the dev environment container.
+2. Select **Get Started**.
+3. Optional: Provide a name for your dev environment.
+4. Choose **Local directory** as the source.
+5. Select **Select** to open the root directory of the code that you would like to work on.
+    A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```
Comment (dvdksn) [inline desktop/dev-environments/create-dev-env.md:127]: Suggestion: add an empty line here to create a separate paragraph for the description part of the instruction

```suggestion
   
   A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```

**Pull request merged**